packages/gatsby-source-contentful/src/download-contentful-assets.js

```diff
@@ -1,6 +1,8 @@
 const ProgressBar = require(`progress`)
 const { createRemoteFileNode } = require(`gatsby-source-filesystem`)
 
+const BATCH_SIZE = 50
```

Comment:
Maybe make this as an adjustable plugin option defaults to 50?

Reply:
Yes, this is perhaps a good idea. Just wanted to check if this fix is beneficial, first. Any thoughts?